s3backgrounddelete/s3backgrounddelete/IEMutil.py

```diff
@@ -49,5 +52,32 @@ def __init__(self, loglevel, eventcode, eventstring):
         self.log_iem()
 
     def log_iem(self):
-        IEM = "IEC:%sS%s:%s" % (self.severity, self.eventCode, self.eventString)
-        syslog.syslog(IEM)
+        self.send(self.module, self.eventCode, self.severity, self.eventString)
+
+    @classmethod
+    def send(self, module, event_id, severity, message_blob):
+        """Send the message."""
+        source = 'S'
+        component= 'S3'
+        EventMessage.init(component, source)
+
+        try:
+            EventMessage.send(module=module, event_id=event_id, severity=severity, message_blob=message_blob)
+        except Exception as exception:
+            msg = ("iem except:%s %s") % (
```

Comment:
ignore exception and continue

Reply:
Ignore exception but add a log message
Take logger as input to constructor.

Reply:
done